camus-etl-kafka/src/main/java/com/linkedin/camus/etl/kafka/mapred/EtlRecordReader.java

```diff
@@ -123,6 +124,8 @@ private CamusWrapper getWrappedRecord(String topicName, byte[] payload) throws I
         CamusWrapper r = null;
         try {
             r = decoder.decode(payload);
+            StatsdReporter statsd = new StatsdReporter(this.context);
+            statsd.logEventDelay(topicName, r);
```

Comment:
Why not move `statsd` initialization to the `EtlRecordReader` initializer (lines 68-70). There you will have the `context`?